amr-wind/incflo_advance.cpp

```diff
@@ -47,15 +47,17 @@ void incflo::pre_advance_stage2()
  *
  * \callgraph
  */
-void incflo::advance()
+void incflo::advance(int inonlin)
 {
     BL_PROFILE("amr-wind::incflo::advance");
+    if (inonlin == 0) m_sim.pde_manager().advance_states();
```

Comment:
I don't understand why the CI isn't complaining about this.

Reply:
looks like we just don't enforce it. Weird.

Reply:
I'll manually add it for now.